iOS Client/PodcastsController/MBPodcastsViewController.swift

```diff
@@ -48,9 +48,9 @@ class MBPodcastsViewController: UIViewController, UITableViewDataSource, UITable
         // swiftlint:enable force_cast
         if indexPath.row < podcasts.count {
```

Comment:
this is fine, but in general I usually live on the edge in the table view methods. I'd rather throw in an `!` than handle some error case that will never happen